cloudformation.template.yml

```diff
@@ -0,0 +1,51 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: 'AWS::Serverless-2016-10-31'
+Description: Delete Workload
+Resources:
+  DeleteWorkloadStepFunction:
+    Type: 'AWS::StepFunctions::StateMachine'
+    Properties:
+      StateMachineName: Delete-Workload-1
+      DefinitionString: !Sub
+        - |-
+          {
+            "Comment": "Delete workload workflow",
+            "StartAt": "DeleteResource",
+            "States": {
+              "DeleteResource": {
+                "Type": "Task",
+                "Resource": "${DeleteResourceLambdaArn}",
+                "End": true,
+                "Retry": [{
+                    "ErrorEquals": ["States.ALL"],
+                    "IntervalSeconds": 1,
+                    "MaxAttempts": 3,
+                    "BackoffRate": 2
+                  }]
+              }
+            }
+          }
+        - DeleteResourceLambdaArn: !GetAtt
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Invalid GetAtt DeleteResourceLambda.Arn for resource DeleteWorkloadStepFunction